js/dom.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
```

Comment:
Try to avoid disabling the eslint checks for the whole file.

Reply:
With out the '/* eslint-disable no-unused-vars */' the rendeeBook() function will give an error since its only called from the index.js

Reply:
I see, when you have a situation that there's no other choice than disabling an eslint rule please do it for only a specific line. Can you try this?